libraries/common/components/Swiper/index.jsx

```diff
@@ -105,28 +107,20 @@ const Swiper = ({
     allowSlidePrev: !disabled,
     allowSlideNext: !disabled,
     onSlideChange: handleSlideChange,
-  }), [
-    autoPlay,
-    additionalModules,
+  }),
+  [additionalModules, classNames.container, classNames.bulletClass, classNames.bulletActiveClass,
+    swiperProps, autoPlay, interval, navigation, showPagination, paginationType, indicators,
     children.length,
-    classNames.bulletActiveClass,
-    classNames.bulletClass,
-    classNames.container,
-    disabled,
-    indicators,
-    interval,
-    navigation,
-    paginationType,
-    showPagination,
-    handleSlideChange,
-    swiperProps,
-  ]);
+    disabled, handleSlideChange]);
 
   return (
     <div className={cls(container, className, 'common__swiper')} aria-hidden={ariaHidden}>
       <OriginalSwiper
         {...internalProps}
         {...swiperProps}
+        aria-live="off"
```

Comment:
assigning the a11y props here basically disables the `A11y` module, and it's not possible to activate it again.